main.tex

```diff
@@ -142,27 +139,28 @@ \section{The Software}
 a common framework that uses the Python \package{requests} package to perform
 HTTP requests to communicate with web services.
 
-For new development, there is a \texttt{template}  that lays out the basic
-framework of any new module.  All modules have a single core
+For new service development, there is a \texttt{template module} that lays
```

Comment:
`module` should not be within the `\texttt{}` segment

Reply:
to clarify, `template` is the name of the module as it appears in the repository; it's technically importable.  "module" is just an English description of that object

Reply:
well, it's called `template_module` isn't it, while the class is `Template`